src/components/FeatureAttributes/fields/FeatureAttributePostProcessField/FeatureAttributePostProcessField.tsx

```diff
@@ -20,9 +20,15 @@ export const FeatureAttributePostProcessField: FC<
   FeatureAttributePostProcessProps
 > = (props) => {
   const { t } = useTranslation(i18n.namespace);
-  const { fieldTextAreaProps } = useContext(FeaturesAttributesContext);
+  const { fieldTextAreaProps, purposes } = useContext(
+    FeaturesAttributesContext,
+  );
   const form = useFormContext();
 
+  if (!purposes.includes("synthesis")) {
```

Comment:
Not sure this has to be limited to just synthesis, its just a post process of a result

Reply:
Marked PostProcess as core